frontend/packages/dev-console/integration-tests/features/topology/quick_search_from_topology_views _to_add_to_project_ODC-2925

```diff
@@ -0,0 +1,73 @@
+Feature: Provide quick search from topology/list views to add to project 
+    As a user, when in Topology List/Graph view, I should be able to have a quick way to search for items to add to my app/project
+
+Background:
+    Given user is at developer perspective
+    And user is "aut-quick-search" namespace
+
+
+@regression
+Scenario: Quick search button in topology chart view
+    Given user is at topology view
+    When user clicks on chart view button
```

Comment:
```suggestion
    When user clicks on graph view button
```

But isn't it a by default view? For this, user has to be in list view first and then he can navigate to graph view.

Reply:
Updated